ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh

```diff
@@ -882,6 +885,9 @@ if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+#export pull-secrets from live cluster for skopeo inspect to use
+run_command "oc extract secret/installation-pull-secrets -n openshift-image-registry --to=/tmp/secret/"
```

Comment:
how about moving it to the block of the specific case?

Reply:
it's better to move it into 60397 only, which will not affect other cases which does not need authfile. `verify_nonhetero` is also called by other cases.

Reply:
@jiajliu moved the logic to case specific and test passed in latest runs.